must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -217,6 +218,15 @@ for ns in $namespaces; do
             done
         done
 
+        # collecting trash list for ceph rbd
+        printf "collecting trash list for ceph rbd \n" | tee -a  "${BASE_COLLECTION_PATH}"/gather-ceph-debug.log
+         # Inspecting ceph block pools for ceph rbd
+        blockpools=$(timeout 60 oc get cephblockpools.ceph.rook.io -n openshift-storage -o jsonpath="{range .items[*]}{@.metadata.name}{'\n'}{end}")
+        COMMAND_OUTPUT_FILE=${COMMAND_OUTPUT_DIR}/rbd_trash_ls
+        for bp in $blockpools; do
+            { timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd trash ls --pool $bp" >> "${COMMAND_OUTPUT_FILE}"; } >> "${COMMAND_ERR_OUTPUT_DIR}"/gather-rbd-rbd_trash_ls-debug.log 2>&1 &
```

Comment:
`gather-rbd-rbd_trash_ls-debug.log` to `gather-rbd_trash_ls-debug.log`

Reply:
still am seeing `gather-rbd-rbd_trash_ls-"${bp}"-debug.log` it should be `gather-rbd-trash_ls-"${bp}"-json-debug.log`? sorry  forgot to ask to add json

Reply:
Will update.